fero/analysis.py

```diff
@@ -911,6 +1016,28 @@ def make_optimization(
                 {"name": "target1", "min": 100, "max": 500}
             ]
 
+        The combination constraints are a list of additional constraints composed of basic arithmetic and boolean operations::
```

Comment:
This comment has sort of gotten out of hand.  Seems like this makes more sense to just talk about all of this in the readme or set up an actual docs page.

Reply:
Yeah - we do go over them in the readme. I'll remove these details here and let the readme speak for itself.